docs/endpoints.rst

```diff
@@ -1,59 +1,56 @@
 
-The first time you run any of the `funcx-endpoint` commands, if there is no existing configuration found at
-`$HOME/.funcx`, you will be prompted to authenticate.  For example, you will likely want to configure an endpoint
-so you may as well start by
+You will be required to authenticate the first time you run `funcx-endpoint`.
+If you have authenticated previously, the endpoint will cache access tokens in
+the local configuration file.
 
-  $ funcx-endpoint configure
+funcX requires authentication in order to associate
+endpoints with users and ensure only authorized users can run tasks on that endpoint. As part of this step
+we request access to your identity and Globus Groups.
 
-You will be asked to authenticate with Globus Auth. We require authentication in order to associate
-endpoints with users and enforce authentication and access control on the endpoint. As part of this step
-we request access to your identity information (to retrieve your email address) and Globus Groups management.
-We use Groups information to facilitate sharing of functions and endpoints by checking the Group membership
-of a group associated with a function.
+To get started you will first want to configure a new endpoint.  ::
 
-Once you've run this command, a directory will be created at `$HOME/.funcx` and a set of configuration files will be generated.
-A default endpoint profile is also created that will be used, whenever you do not explicitly
-specify a profile to use for endpoint actions.
+  $ funcx-endpoint configure
 
-You can also set up auto-completion for the `funcx-endpoint` commands in your shell, by using the command
+Once you've run this command, a directory will be created at `$HOME/.funcx` and a set of default configuration files will be generated.
 
-  $ funcx-endpoint --install-completion [zsh bash fish ...]
+You can also set up auto-completion for the `funcx-endpoint` commands in your shell, by using the command ::
 
-Which will allow commands and endpoint names to autocomplete.
+  $ funcx-endpoint --install-completion [zsh bash fish ...]
 
-Configuring funcX
------------------
 
-A configuration file will be created at `$HOME/.funcx/config.py`. This contains
-base information regarding the endpoint, such as a username and email. By default, this includes
-an address and port for a local broker, which is used if a local broker is deployed.
-You can also set the address of the endpoint for your workers to connect to.
-This is necessary if your workers are not deployed on the
-same resource as the endpoint (e.g., when using a batch submission system, or cloud workers).
+Configuring an Endpoint
+----------------------------
 
-.. note:: If your funcX workers are not deployed on the same resource as the endpoint you must set the endpoint address for the workers to find the endpoint. This is done by setting the endpoint_address.
+After creating an endpoint, a configuration file will be created at `$HOME/.funcx/config.py`. This file contains
```

Comment:
@kylechard I think we should remove this bit completely. We no longer rely on this file.